Gemfile

```diff
@@ -60,6 +60,11 @@ gem 'bootstrap-typeahead-rails'
 # for authorization layer
 gem 'pundit'
 
+# for analytics
+gem 'ahoy_matey'
+
+gem 'mysql2'
```

Comment:
We are not using mysql anymore

Reply:
Rails raised an error and said I need to add this to Gemfile. Without this, the server would not start. Do I need to make some change to my local database.yml?